ha/core/system_health/system_health_exception.py

```diff
@@ -21,3 +21,12 @@ class HealthNotFoundException(HaSystemHealthException):
 
 class InvalidElement(HaSystemHealthException):
     """Exception to indicate health missing for element."""
+
+class InvalidAction(HaSystemHealthException):
```

Comment:
Action handler is not part of the system health module, please move these to appropriate module.

Reply:
Done, moved to event_manager/error.py